modules/openthread/Kconfig.thread

```diff
@@ -184,7 +184,7 @@ config OPENTHREAD_DEFAULT_TX_POWER
 
 config OPENTHREAD_BLE_TCAT_THREAD_STACK_SIZE
 	int "Openthread default TCAT stack size"
-	default 4200
+	default 5120
```

Comment:
Consider adding here the 'if PSA' clause.

Reply:
fixed